libbeat/common/config.go

```diff
@@ -190,6 +202,39 @@ func (c *Config) SetChild(name string, idx int, value *Config) error {
 	return c.access().SetChild(name, idx, value.access(), configOpts...)
 }
 
+func (c *Config) IsDict() bool {
+	return c.access().IsDict()
+}
+
+func (c *Config) IsArray() bool {
+	return c.access().IsArray()
+}
+
+func (c *Config) DebugString() string {
```

Comment:
This could be called just "String()" as it is the String output of loaded config. Should this be part of ucfg directly?